command/action.go

```diff
@@ -100,6 +93,8 @@ func (c *ActionCommand) AutocompleteArgs() complete.Predictor {
 
 func (c *ActionCommand) Name() string { return "action" }
 
+const defaultEscapeChar = "~"
+
 func (c *ActionCommand) Run(args []string) int {
```

Comment:
Since we're just wrapping another command for now, I don't think we need to do any logic here, something like this would be enough:

```go
func (c *ActionCommand) Run(args []string) int {
	cmd := &JobActionCommand{
		Meta:   c.Meta,
		Stdin:  c.Stdin,
		Stdout: c.Stdout,
		Stderr: c.Stderr,
	}

	return cmd.Run(args)
}
```

This also means we can get rid of all flags and have the`AutocompleteFlags` logic.

```go
func (c *ActionCommand) AutocompleteFlags() complete.Flags {
	return nil
}
```

Reply:
Oh neat! I didn't expect this would pass the flags (-group -task -t etc.) along automatically.